website/content/docs/connect/config-entries/api-gateway.mdx

```diff
@@ -329,3 +397,166 @@ Specifies the Enterprise [admin partition](/consul/docs/enterprise/admin-partiti
+
+#### Values
+
+- Default: None
+- Data type: List of maps
+
+The following table describes the parameters you can specify in a member of the `Providers` list:
+
+| Parameter | Description | Data type | Default |
+| ---       | ---         | ---       | ---     |
+| `Name` | Specifies the name of the provider. | String | None |
+| `VerifyClaim` | Specifies a list of paths and a value that define the claim. Consul verifies requests that match the claims declared in the listener JWT configuration and allow the request through the gateway. The `VerifyClaims` map specifies the following settings: <li>`Path`: List of paths to the claims</li><li>`Value`: ???</li> | Map | None |
+
+Refer to [Configure JWT verification settings](#configure-jwt-verification-settings) for an example configuration. 
+
+### `Listeners[].override`
+
+Specifies a block of configurations to apply to the gateway listener. The override settings have  precedence over the [`default` configurations](#listeners-default).
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+### `Listeners[].override{}.JWT`
+
+Specifies a block of JWT verification configurations to apply to the gateway listener. The override settings have  precedence over the [`default` configurations](#listeners-default), as well as any route-specific JWT configurations.
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+### `Listeners[].override{}.JWT{}.Providers`
+
+Specifies a list of JWT provider configurations to apply to the gateway listener. A provider configuration contains the name of the provider and claims. The override settings have precedence over defaults as well as any listener specific configuration. 
+
+#### Values
+
+- Default: None
+- Data type: List of maps
+
+The following table describes the parameters you can specify in a member of the `Providers` list:
+
+| Parameter | Description | Data type | Default |
+| ---       | ---         | ---       | ---     |
+| `Name` | Specifies the name of the provider. | String | None |
+| `VerifyClaim` | Specifies a list of paths and a value that define the claim. Consul verifies requests that match the claims declared in the listener JWT configuration and allow the request through the gateway. The `VerifyClaims` map specifies the following settings: <ul><li>`Path`: List of paths to the claims</li><li>`Value`: ???</li></ul> | Map | None |
+
+Refer to [Configure JWT verification settings](#configure-jwt-verification-settings) for an example configuration. 
+
+## Examples
+
+Add the following example to the Examples section.
+
+### Configure JWT verification settings
+
+The following example configures `listener-one` to verify that requests are from clients that have Okta user permissions by default. The listener also verifies claims that  <TODO - need help describing the example>:
```

Comment:
```suggestion
The following example configures `listener-one` to verify that requests include a token with Okta user permissions by default. The listener also verifies that the token has an audience of `api.apps.organization.com`.
```

It would probably also be helpful to describe that the user role verification can be overridden at the route level because it is only a `default`. The route could itself include a `JWTAuth` filter that verifies an Okta `admin` role instead.

In contrast, the audience claim verification will happen for every single request that comes through the listener, and it cannot be overridden at the route level because it is an `override` at the gateway level.

The order of precedence being
1. Gateway `override`
2. HTTPRoute filter
3. Gateway `default`